concepticondata/conceptlists.tsv

```diff
@@ -258,4 +258,5 @@ Backstrom-1992-210d	Backstrom, Peter C. and Radloff, Carla F.	1992	d	201	basic,a
\ No newline at end of file
+Luniewska-2019-299	Łuniewska, Magdalena AND Wodniecka, Zofia AND Miller, Carol A. AND Smolík, Filip AND Butcher, Morna AND Chondrogianni, Vasiliki AND Hreich, Edith Kouba AND Messarra, Camille AND A. Razak, Rogayah AND Treffers-Daller, Jeanine AND Yap, Ngee Thai AND Abboud, Layal AND Talebi, Ali AND Gureghian, Maribel AND Tuller, Laurice AND Haman, Ewa	2019		299	acquisition	Afrikaans, AmericanEnglish, Catalan, Czech, Danish, Dutch, English, Finnish, Gaelic, German, Greek, Hebrew, Hungarian, Icelandic, Irish, Xhosa, Italian, LebaneseArabic, Lithuanian, Luxembourgish, Malay, Maltese, Persian, Polish, Russian, SouthAfricanEnglish, Serbian, Slovak, Spanish, Swedish, Turkish, WesternArmenian	Global	https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0220611	Luniewska2019		An updated version of [Luniewska-2016-299](:ref:Luniewska-2016-299), including seven additional languages with respective age of acquisition information. This version also includes higher precision numbers for the acquisiton variables.		
+Pereira-2018-180	Pereira, Francisco and Lou, Bin and Pritchett, Brianna and Ritter, Samuel and Gershman, Samuel J. and Kanwisher, Nancy and Botvinick, Matthew and Fedorenko, Evelina	2018		180	specific	English	English	https://www.nature.com/articles/s41467-018-03068-4	Pereira2018		The given list was compiled in the frameworks of research on the neural basis of semantic representations. As reported by the authors, it resulted from clustering words from [the massive text corpora](:bib:Brysbaert2014) into 180 semantic groups, covering the entire semantic space. The most representative concepts, manually selected from each cluster, were then used as input stimuli in three fMRI experiments in order to investigate the relationship between the linguistic representations and the resulting imaging data. The researchers report that semantic information is not confined to a specific brain region, as commonly assumed, but rather distributed across several high-level cortical networks.	1-13	
```

Comment:
`[the massive text corpora]` -> `[massive text corpora]` no article here. `the entire semantic space` -> `the entire semantic space of the corpus`.